src/Calendar/modes/Days.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { getDaysInMonth, setDate, isSameDay } from 'date-fns'
+
+import { StyledWeekDays, StyledDays, StyledTimeUnit } from '../styled'
+
+export const WeekDays = () => (
+  <StyledWeekDays>
+    <li>Mo</li>
+    <li>Tu</li>
+    <li>We</li>
+    <li>Th</li>
+    <li>Fr</li>
+    <li>Sa</li>
+    <li>Su</li>
+  </StyledWeekDays>
+)
+
+function getRenderDay (value: Date, browseDate: Date, onChange: Function) {
+  return (_: undefined, index: number) => {
+    const day = index + 1
+    const thisDay = setDate(browseDate, day)
+    const isActive = isSameDay(value, thisDay)
+
+    return (
```

Comment:
Maybe this can be extracted and reused throughout the other getRenders since everything other than the onClick and StyledTimeUnit children is identical.